### DISCUSSION TITLE
Did eslint have support for fixing missing imports?

### DISCUSSION THREAD

**Discussion by croraf** [eslint/eslint#15004] [General]
I had "Add missing imports" in quick fix options of VSCode but they are missing since mid 2021.

So I'm wondering:
- did ESLint stop supporting this, or 
- there is some bug introduced two months ago, or 
- this was never a feature provided by eslint?

**Comment by mdjermanovic** [comment]
> I had "Add missing imports" in quick fix options of VSCode but they are missing since mid 2021.

Hi @croraf!

This was never a feature provided by core ESLint or any ESLint core rules.

It could be a feature provided by a third-party plugin (e.g., a suggestion provided by a third-party rule), or another tool unrelated to ESLint, or a VSCode built-in feature.
